features/upgrade/samples_operator/upgrade.feature

```diff
@@ -37,3 +37,99 @@ Feature: image-registry operator upgrade tests
     Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Available')['status'] == "True"
     Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Progressing')['status'] == "False"
     Given the expression should be true> cluster_operator('openshift-controller-manager').condition(type: 'Degraded')['status'] == "False"
+
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: Upgrade imagestream from old version which not implement node credentials - prepare 
+    Given I switch to cluster admin pseudo user
+    When I run the :extract admin command with:
+      | resource  | secret/pull-secret |
+      | namespace | openshift-config   | 
+      | to        | /tmp               |
+      | confirm   | true               |
+    Then the step should succeed
+    When I run the :new_project client command with:
+      | project_name | devexp-p1 |
+    Then the step should succeed
+    When I use the "devexp-p1" project
+    When I run the :create_secret client command with:
+      | secret_type | generic                                  |
+      | name        | pj-secret                                |
+      | from_file   | .dockerconfigjson=/tmp/.dockerconfigjson |
+      | type        | kubernetes.io/dockerconfigjson           |
+    Then the step should succeed
+    When I run the :import_image client command with:
+      | from             | registry.redhat.io/rhscl/ruby-25-rhel7:latest |
+      | confirm          | true                                          |
+      | image_name       | imagestream1:latest                           |
+      | reference-policy | local                                         |
+    Then the step should succeed
```

Comment:
The project is include pull secret, still import failed?

Reply:
@wewang58 this is a upgrade prepare case on 4.4 which is not include the pull secret.